### PR TITLE
remove legacy code no longer used

### PR DESCRIPTION
**Pull request by coblej**

### CODE REVIEW

**Comment by coblej** [comment]
Also pins Blacklight to 5.15.0 to avoid problem encountered with Blacklight 5.16.3 on Travis.

File: ddr-models.gemspec [hunk 1]
```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib", "app/models"]
 
   s.add_dependency "rails", "~> 4.1.13"
+  s.add_dependency "blacklight", "5.15.0"
```
Comment (dchandekstark) [inline ddr-models.gemspec]: Shouldn't we change the development dependency instead?

**Comment by coblej** [comment]
@dchandekstark Yeah, probably.  I'll change it.

**Comment by dchandekstark** [comment]
@coblej The build failure is due to a bug in ActiveFedora (not explicitly requiring rdf/vocab), so I'm willing to merge as is.

**Pull request merged**